### PR TITLE
Harvester assets

### PR DESCRIPTION
**Pull request by Hamms**
Adds new harvester assets. Also replace "bean" in code with "lettuce"

Before

image

After

image

### CODE REVIEW

**Comment by joshlory** [comment]
Nit: should the corn baseline be moved up so it's in line with the other two?

File: apps/src/maze/api.js [hunk 1]
```diff
@@ -312,8 +312,8 @@ exports.getPumpkin = API_FUNCTION(function (id) {
   Maze.subtype.getPumpkin(id);
 });
 
-exports.getBean = API_FUNCTION(function (id) {
-  Maze.subtype.getBean(id);
+exports.getLettuce = API_FUNCTION(function (id) {
+  Maze.subtype.getLettuce(id);
```
Comment (joshlory) [inline apps/src/maze/api.js:316]: I realize this is changing an existing name, but I wonder if these APIs should take the vegetable as a parameter?  E.g. `exports.harvest = API_FUNCTION(Maze.subtype)` then `harvest(LETTUCE, id)`.

Comment (Hamms) [reply apps/src/maze/api.js:316]: I'd say the primary consideration for the API is what it makes the user's code look like; although it ends up being more work for us this way, the resulting code that the kids see is much nicer.

Comment (joshlory) [reply apps/src/maze/api.js:316]: I see, this translates directly to the blocks API.

**Comment by joshlory** [comment]
LGTM

**Pull request merged**